bot.testnet.toml

```diff
@@ -0,0 +1,27 @@
+propose-time-window = 240
+challenge-time-window = 3
+l2-starting-number = $OPBNB_BRIDGE_BOT_L2_STARTING_NUMBER
+
+[rpcs]
+l1-rpc = "$OPBNB_BRIDGE_BOT_L1_RPC"
+l2-rpc = "$OPBNB_BRIDGE_BOT_L2_RPC"
+
+[tx-signer]
+privkey = "$OPBNB_BRIDGE_BOT_PRIVKEY"
+gas-price = 9000000000
+
+[db]
+host = "$OPBNB_BRIDGE_BOT_DB_HOST"
+port = $OPBNB_BRIDGE_BOT_DB_PORT
+user = "$OPBNB_BRIDGE_BOT_DB_USER"
+password = "$OPBNB_BRIDGE_BOT_DB_PASSWORD"
+name = "$OPBNB_BRIDGE_BOT_DB_NAME"
+
+[l1-contracts]
+optimism-portal                 = "0x4386c8abf2009ac0c263462da568dd9d46e52a31"
+l2-output-oracle                = "0xff2394bb843012562f4349c6632a0ecb92fc8810"
+l1-cross-domain-messenger       = "0xd506952e78eecd5d4424b1990a0c99b1568e7c2c"
+
+[l2-standard-bridge-bot]
+contract-address                = "0x62fa6549e240076D466E1150ce587905c55f12F1"
+log-filter-block-range = 1000
```

Comment:
plz pre-config the whitelist according to https://github.com/bnb-chain/opbnb-bridge-tokens#opbnb-testnet-token-list